Yosemite/Yosemite/Stores/WooShippingStore.swift

```diff
@@ -296,6 +296,7 @@ private extension WooShippingStore {
                                   address: WooShippingDestinationAddress,
                                   completion: @escaping (Result<WooShippingDestinationAddressUpdate, Error>) -> Void) {
         remote.updateDestinationAddress(siteID: siteID, orderID: orderID, address: address, completion: completion)
+        setLastModifiedDateForOrder(siteID: siteID, orderID: orderID)
```

Comment:
Nit: I think we should update the modified date only when the destination address update is successful.